txgh/spec/config/key_manager_spec.rb

```diff
@@ -85,10 +104,4 @@
       expect(KeyManager.project_names).to eq([project_name])
     end
   end
-
-  describe '#repo_names' do
```

Comment:
What happened to this test?

Reply:
There was no point of replicating this method in gitlab classes as it is never used even for github. Does it have any sense of keeping it?